kr/kr_pgp.go

```diff
@@ -322,3 +324,98 @@ func uninstallCodesigning() {
 	exec.Command("git", "config", "--global", "--unset", "commit.gpgSign").Run()
 	exec.Command("git", "config", "--global", "--unset", "tag.forceSignAnnotated").Run()
 }
+
+func pgpSignCommand(c *cli.Context) (err error) {
+	message := c.String("message")
+	sigType := c.String("sigType")
+	if sigType != "detach" && sigType != "attach" && sigType != "clearsign" {
+		err = errors.New("Invalid signature type!\n")
+		os.Stderr.WriteString(err.Error())
+		return
+	}
+
+	request, err := kr.NewRequest()
+	if err != nil {
+		os.Stderr.WriteString(err.Error())
```

Comment:
If there is a fatal error we can use `PrintFatal(..)` to set a bad exit code as well